package.json

```diff
@@ -10,17 +10,29 @@
     "preinstall": "",
     "postinstall": "gulp build",
     "start": "nf run gulp dev",
-    "test": "gulp test",
+    "test": "gulp test",    
+    "changelog": "gulp changelog",
+    "updatedocs": "gulp docs",
+    "preversion": "npm run test",
+    "version" : "npm version && npm run changelog && npm run updatedocs",
+    "postversion": "npm run commit && npm run push",
+    "commit": "git add .",
+    "push": "git push",
+    "version:major": "npm version major",
+    "version:minor": "npm version minor",
+    "version:patch": "npm version patch",
```

Comment:
Hm. I realized the hooks will be called despite the specified type of version we're upgrading. Essentially, we can delete lines 21, 22, 23.